workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,180 @@
+author: '@giridhar7632'
+---
+
+# Parallax Effect
+
+One of the most popular effects of modern web experience is **the parallax effect**. We are going to build a basic project to understand how the parallax effect works.
+To understand this workshop you will need to be familiar with HTML, CSS and JavaScript.
+
+<a href="https://total-parallax.giridharhackclu.repl.co/">![Final Result](https://cloud-1z03ndewk.vercel.app/ezgif.com-gif-maker__2_.gif)</a>
+
+Here's the [live demo](https://total-parallax.giridharhackclu.repl.co/), and [source code](https://repl.it/@Giridharhackclu/total-parallax#index.html) of the project.
+
+## Parallax
+
+Parallax movement is when things move at different speeds relative to each other. This effect can be used to create an illusion of depth.
+We will experience this effect in websites while scrolling a webpage. It's called Parallax scrolling.
+
+The technique is popular in many places including video games, where it’s usually seen in the layered backgrounds.
+
+But what exactly causes the illusion?
+
+We know that objects at farther distances move with slower speeds than the nearer ones. As you scroll or do something to trigger the effect, you experience the illusion as elements move with different velocities.
+
+> **Note:** Parallax scrolling does not always work on mobile devices smartphones. However, you can use media queries to turn off the effect on mobile devices.
+
+We can see how the parallax scrolling works in this example [here](https://codepen.io/samdbeckham/full/OPXPNp).
+
+<a href="https://codepen.io/samdbeckham/full/OPXPNp" target="_blank"><img src="https://cloud-kt149rdr8.vercel.app/ezgif.com-add-text.gif" alt="parallax scrolling"/></a>
+
+^As you scroll, each of the layers moves at different speeds.
+
+## Advantages of Parallax Effect
+
+If you want to make your website amazing, it's not enough to show static elements. You have to amaze your visitors and make them want to explore your site, otherwise they can leave fast. You have to show them some cool things and some short visual surprises. The Parallax effect makes it possible. Moving components on scrolling or on hovering will make them interested.
+You can create something interesting using parallax effect of your choice.
+
+## Getting started
+
+Practically a parallax is composed of more than one layer in parallel, moving along on scroll at different speeds, giving us the feeling that they're at different distances.
+
+### Setup
+
+We will use [Repl.it](https://repl.it) to build this project.
+
+> Repl.it is an online code editor. It's similar to Google Docs, but has some important features that make it much better for typing code than a regular text editor.
+
+Fork this repl and explore! [here](https://repl.it/@Giridharhackclu/parallax-starter#index.html).
+
+It contains three `div` elements with classes `layer l1`, `layer l2` and `layer l3` added with some basic styles. Go ahead! use your creativity and customise the styles.
```

Comment:
> Go ahead! use your creativity and customise the styles.

This is a great prompt! But at this stage of the workshop, the person doing the workshop barely knows what’s going on since we’re just at the beginning, so I think this prompt is best saved for the end.

Reply:
okay!